shell/platform/android/io/flutter/embedding/android/FlutterFragment.java

```diff
@@ -90,20 +93,26 @@ public static FlutterFragment newInstance() {
    * @param appBundlePath the path to the app bundle which contains the Dart app to execute, defaults
    *                      to {@link FlutterMain#findAppBundlePath(Context)}
    * @param flutterShellArgs any special configuration arguments for the Flutter engine
+   * @param renderMode render Flutter either as a {@link FlutterView.RenderMode#surface} or a
+   *                   {@link FlutterView.RenderMode#texture}. Use {@code texture} for situations
+   *                   where this {@code FlutterFragment} might have content on top of it, such as
+   *                   a drawer.
```

Comment:
nit: It's hard to get into the details a ton in just a line or two here but as is this isn't letting people know that using `texture` most likely comes with a huge performance cost.  I'd probably rephrase to something along the lines of "Use `{@code texture}` when you're willing and able to sacrifice performance to have this `{@code FlutterFragment}` interleave with other Android views. `{@code surface}` is a much more performant default, but won't composite with complicated View hierarchies." Exact wording doesn't matter much to me but I think a really rough summary of the pros/cons would be worth it here.

Reply:
done